index.html

```diff
@@ -44,14 +44,19 @@ <h3 class="description"><a href="https://servo.org/"><b>Servo</b></a>
         <div class="col-sm-8 col-lg-4 col-sm-push-2 col-lg-push-0">
             <a class="btn btn-primary btn-large" role="button"
                href="nightly/windows-msvc/servo-latest.exe">Windows Build Installer (64-bit)</a><br>
+
             <a class="btn btn-primary btn-large" role="button"
                href="nightly/windows-msvc/servo-latest.zip">Windows Build (64-bit)</a><br>
+
             <a class="btn btn-primary btn-large" role="button"
                href="nightly/uwp/servo-latest.zip">UWP Build</a><br>
+
             <a class="btn btn-primary btn-large" role="button"
                href="nightly/mac/servo-latest.dmg">macOS Build</a><br>
+
             <a class="btn btn-primary btn-large" role="button"
                href="nightly/linux/servo-latest.tar.gz">Linux Build (64-bit)</a><br>
+
```

Comment:
What if instead we added `<a href=".../servo-latest.foo.sha256">md5</a>` to each of lines instead?

Reply:
I tried that, but it looked gross. I'll drop a screenshot here.

Probably there's a better way to do it, but I'm not sure.

Reply:
The issue is that the text is not centered:

![image](https://user-images.githubusercontent.com/37223377/81991502-3ad8c680-95f6-11ea-8815-a994d1f89859.png)

Reply:
Maybe I could use a table?

Reply:
<img width="319" alt="Screen Shot 2020-05-14 at 6 28 43 PM" src="https://user-images.githubusercontent.com/27658/81992095-c19a9d00-9610-11ea-83f2-e4ac35ff30d4.png">
This might be reasonable. I got that with:
```html
<span class="col-sm-4 col-sm-push-0"><a class="btn btn-primary btn-large" role="button" href="nightly/windows-msvc/servo-latest.exe">Windows Build Installer (64-bit)</a><br><a class="btn btn-primary btn-large" role="button" href="nightly/windows-msvc/servo-latest.zip">Windows Build (64-bit)</a></span>
<span class="col-sm-push"><a class="btn">sha256</a><br><a class="btn">sha256</a></span>
```

Reply:
Looks nice! I'll add that.

Reply:
Huh. Doesn't work on my end:

![image](https://user-images.githubusercontent.com/37223377/81994762-6d39f200-95fd-11ea-9649-ce6b22e8f23c.png)

With code:

```html
<div class="col-sm-8 col-lg-4 col-sm-push-2 col-lg-push-0">
            <span class="col-sm-4 col-sm-push-0">
                <a class="btn btn-primary btn-large" role="button"
                   href="nightly/windows-msvc/servo-latest.exe">Windows Build Installer (64-bit)</a><br>

                <a class="btn btn-primary btn-large" role="button"
                   href="nightly/windows-msvc/servo-latest.zip">Windows Build (64-bit)</a><br>

                <a class="btn btn-primary btn-large" role="button"
                   href="nightly/uwp/servo-latest.zip">UWP Build</a><br>

                <a class="btn btn-primary btn-large" role="button"
                   href="nightly/mac/servo-latest.dmg">macOS Build</a><br>

                <a class="btn btn-primary btn-large" role="button"
                   href="nightly/linux/servo-latest.tar.gz">Linux Build (64-bit)</a><br>
            </span>
            <span class="col-sm-push">
                <a class="btn" href="nightly/windows-msvc/servo-latest.exe.sha256">sha256</a><br>

                <a class="btn" href="nightly/windows-msvc/servo-latest.zip.sha256">sha256</a><br>

                <a class="btn" href="nightly/uwp/servo-latest.zip.sha256">sha256</a><br>

                <a class="btn" href="nightly/mac/servo-latest.dmg.sha256">sha256</a><br>

                <a class="btn" href="nightly/linux/servo-latest.tar.gz.sha256">sha256</a><br>
            </span>
        </div>
```

Reply:
Ah, fixed it! Changed line 2 from `col-sm-4` to `col-sm-8`. I'm pretty inexperience with bootstrap and especially its grids...

Doesn't look great on mobile, but no one will be downloading a binary on their phone, so it's fine.